Slash/Utility/Comments/Comments.pm

```diff
@@ -1317,10 +910,7 @@ sub displayThread {
 		|| $user->{mode} eq 'child') {
```

Comment:
Shows up in printCommentsMain;misc;default but I've no idea if the statement it's even can evaluate to anything but true. No biggie though. It's not hurting anything if it really isn't used or causing a massive slowdown, so it can stay for now.